packages/office-ui-fabric-react/src/components/Panel/Panel.types.ts

```diff
@@ -200,7 +200,7 @@ export interface IPanelProps extends React.HTMLAttributes<PanelBase> {
   /**
    * Optional custom function to handle clicks outside this component
    */
-  onOuterClick?: () => void;
+  onOuterClick?: () => boolean | void | undefined;
```

Comment:
Why add undefined here? Not saying it's wrong, just wondering

Reply:
Wanted to encompass more return values. Though I guess the return type will probably be either a boolean or void, so that is probably unnecessary.

Reply:
Yeah, you'll want to make the minimum change possible to the surface here. Once we add a return type here we can't remove it without a breaking change (and major release)